code/modules/mob/dead/observer/observer.dm

```diff
@@ -1235,7 +1235,10 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 			. += ""
 
 	if(timeofdeath)
-		. += "Time Since Death: [duration2text_sec(world.time - timeofdeath)]"
+		var/time_since_death = world.time - timeofdeath
+		var/format = (time_since_death >= 1 HOURS ? "hh:mm:ss" : "mm:ss")
+
+		. += "Time Since Death: [gameTimestamp(format, time_since_death)]"
```

Comment:
I don't think you want to use gameTimestamp instead of just time2text because I don't think we want the timezone accounted for like gameTimestamp does. All other game calculations against timeofdeath are just against world.time.

Also your example video doesn't show the counter moving any when it should be growing every second time passes.

You can simply right click your mob, view variables, and edit your time of death value to a low number (meaning far in the past including negative) for testing purposes rather than needing to code a temporary debugging verb for this.

Reply:
Oh for some reason I didn't think you could VV edit something to a negative value. That definitely would've made testing a lot easier.